source/elements/oneMKL/source/domains/spblas/operations/spmv.rst

```diff
@@ -0,0 +1,368 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_spmv_header:
+
+spmv
+====
+
+Computes a sparse matrix by dense vector product.
+
+.. rubric:: Description and Assumptions
+
+The ``oneapi::mkl::sparse::spmv`` routine computes a sparse matrix by dense
+vector product defined as:
+
+.. math::
+
+   y \leftarrow \alpha \cdot \text{op}(A) \cdot x + \beta \cdot y
+
+
+| where:
+|   :math:`\alpha` and :math:`\beta` are scalars,
+|   :math:`x` is a dense vector of size ``n``,
+|   :math:`y` is a dense vector of size ``m``,
+|   :math:`\text{op}(A)` is a transformed sparse matrix of size ``m``-by-``n``,
+|   :math:`\text{op}()` is the transform operation using the following description:
+
+.. math::
+
+    \text{op}(A) = \begin{cases} A,& \text{oneapi::mkl::transpose::nontrans}\\
+                                 A^\mathsf{T},& \text{oneapi::mkl::transpose::trans}\\
+                                 A^\mathsf{H},& \text{oneapi::mkl::transpose::conjtrans}
+                   \end{cases}
+
+.. _onemkl_sparse_spmv_descr:
+
+spmv_descr
```

Comment:
I don't see the purpose of the spmv_descr listed anywhere here,  what is it for ?  why do we need it ?

Reply:
I added a short global note in https://github.com/uxlfoundation/oneAPI-spec/pull/522/commits/ac1b22388b3615ed559f753105e395a937f2033b. I don't think the spec needs to justify every design choice though.

Reply:
ok, that comment is fine, I would recommend to put a small sentence for each descr definition (there are three now) saying something like 

"spmv_descr object persists through the various stages of the spmv operation to house relevant state, optimizations and workspaces."

Reply:
I am not keen on duplicating that comment for every operation as:

1. I expect this will apply to every operation. Duplication makes the spec tiresome to read so is best to avoid.
2. I don't think this is important enough to be repeated multiple times. The user does not need this information to safely use the API.

Reply:
While I do understand Romain's concerns, I agree with Spencer here.

Reply:
Alright I'm outvoted so I applied your suggestion in https://github.com/uxlfoundation/oneAPI-spec/pull/522/commits/8f94f4dfbbddeb4880b04e83cce6470043218c74